postgresql/hooks/init

```diff
@@ -20,7 +20,7 @@ if [[ ! -f "{{pkg.svc_data_path}}/PG_VERSION" ]]; then
     -E utf8 \
     -D {{pkg.svc_data_path}} \
     --pwfile {{pkg.svc_config_path}}/pwfile \
-    --locale POSIX
+    --locale POSIX {{ #if cfg.data_checksums }}--data-checksums{{ /if }}
```

Comment:
From reading a bit about this setting I would say we have it on by default or? It protects against data corruption but doesn't seem to have any downside to it.